utils/bundles.stanza

```diff
@@ -146,9 +146,12 @@ public pcb-bundle spi  (pins:Tuple<SPIPins>):
       SPI-SDIO  : make-pin(`sdio)
 
 ; Most common spi bundle
-public defn spi ():
+public defn spi-peripheral :
   spi([SPI-SDO SPI-SDI SPI-CS])
 
+public defn spi-controller :
```

Comment:
likewise about unit tests, if possible

Reply:
added usb-accel.stanza to test spi bundles